### PR TITLE
PD-149 SCALE 22.12.2/23.10 App Article Improvements

### PR DESCRIPTION
**Pull request by tonyriv3**

### CODE REVIEW

File: content/SCALE/SCALETutorials/Apps/Customizing Advanced (Kubernetes) Settings/Configuring Host Path Safety Checks.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+{{< toc >}}
+
+TrueNAS SCALE uses host path safety checks to ensure that host path volumes are secure when creating apps. We recommend creating datasets for applications that do not share the same host path as an SMB or NFS share.
+
+Since TrueNAS considers shared host paths non-secure, apps that use shared host paths (such as an SMB shared dataset) fail to deploy. 
+
+## Using Shared Host Paths with Safety Checks Enabled
+
+If you group share and application data under a common dataset (such as *media*) where both use a path such as */tank/media/*, the application will fail to deploy.
```
Comment (micjohnson777) [inline content/SCALE/SCALETutorials/Apps/Customizing Advanced (Kubernetes) Settings/Configuring Host Path Safety Checks.md:18]: present tense active voice change suggestion:
```suggestion
If you group share and application data under a common dataset (such as *media*) where both use a path such as */tank/media/*, the application fails to deploy. 
```

File: content/SCALE/SCALETutorials/Apps/Customizing Advanced (Kubernetes) Settings/_index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: "Customizing Advanced (Kubernetes) Settings"
+geekdocCollapseSection: true
+weight: 55
+---
+
+The Kubernetes Settings screen allows users to customize network, system, and cluster settings for all apps in TrueNAS SCALE.
```
Comment (micjohnson777) [inline content/SCALE/SCALETutorials/Apps/Customizing Advanced (Kubernetes) Settings/_index.md:7]:
```suggestion
The **Kubernetes Settings** screen allows users to customize network, system, and cluster settings for all apps in TrueNAS SCALE.
```

File: content/SCALE/SCALETutorials/Apps/UsingApps.md [hunk 3]
```diff
@@ -78,7 +65,7 @@ To deploy a custom application container in the SCALE web interface, go to **App
 
 Custom applications use the system-level Kubernetes Node IP settings by default. You can assign an external interface to custom apps by setting one on the **Networking** section of the **Launch Docker Image** form. 
 
-Unless you need to run an application separately from the Web UI, we recommend using the default Kubernetes Node IP (0.0.0.0) to ensure apps function properly.
+Unless you need to run an application separately from the Web UI, we recommend using the default Kubernetes Node IP (0.0.0.0) to ensure apps function correctly.
```
Comment (micjohnson777) [inline content/SCALE/SCALETutorials/Apps/UsingApps.md:68]: I think you are specifically talking about the UI setting so suggesting this change:
```suggestion
Unless you need to run an application separately from the Web UI, we recommend using the default Kubernetes **Node IP** (0.0.0.0) to ensure apps function correctly.
```

Comment (micjohnson777) [reply content/SCALE/SCALETutorials/Apps/UsingApps.md:68]: If you are not intending to specifically mention the UI setting, then change Node IP to node IP

File: content/SCALE/SCALEUIReference/Apps/AppsScreensSCALE.md [hunk 4]
```diff
@@ -216,11 +222,11 @@ The **Available Applications** screen displays the widgets for all applications
 
 The **Install** button on each application card opens the configuration wizard for that application.
 
-Click on the application icon or name to open an ***appname* Application Summary** window that includes information on the **Catalog**, **Categories**, **Train**, **Status** and **Versions** for that application.
+Click on the application icon or name to open an **appname** Application Summary** window that includes information on the **Catalog**, **Categories**, **Train**, **Status**, and **Versions** for that application.
```
Comment (micjohnson777) [inline content/SCALE/SCALEUIReference/Apps/AppsScreensSCALE.md:225]: The bold emphasis is off here, what part of this is bold and on the UI screen? if one is a variable and the other is static on the UI screen, then make this change
```suggestion
Click on the application icon or name to open an ***appname* Application Summary** window that includes information on the **Catalog**, **Categories**, **Train**, **Status**, and **Versions** for that application.
```

**Review by clairewrtr — COMMENTED** [review]
Looks good.

**Review by micjohnson777 — CHANGES_REQUESTED** [review]
Excellent!! The Apps redesign in Cobia will completely change all this content so most excellent updates!

File: content/SCALE/SCALETutorials/Apps/Customizing Advanced (Kubernetes) Settings/Configuring Host Path Safety Checks.md (hunk 1 above)
Comment (tonyriv3) [reply content/SCALE/SCALETutorials/Apps/Customizing Advanced (Kubernetes) Settings/Configuring Host Path Safety Checks.md:18]: Forgot to `ctrl + f` for "will"!

**Pull request merged**